lib/rworkflow/sidekiq_helper.rb

```diff
@@ -3,35 +3,14 @@
 module Rworkflow
   module SidekiqHelper
     def self.included(klass)
-      klass.send :extend,  ClassMethods
-      klass.send :include, InstanceMethods
+      klass.send :extend, ClassMethods
```

Comment:
if we have a hard dependency on Rails, you can just use ActiveSupport::Concern no?